app/src/test/java/com/example/task1/data/banners/BannersTest.kt

```diff
@@ -63,54 +63,54 @@ class BannersTest {
     @Test
     fun fetchBannersFromAPI() = testScope.runTest {
```

Comment:
Add comments to explain what this function is doing in each step. 
Do the same for all test cases.